apps/web/pages/teams/index.tsx

```diff
@@ -11,19 +12,28 @@ import PageWrapper from "@components/PageWrapper";
 
 function Teams() {
   const { t } = useLocale();
+  const session = useSession();
+
+  const NewTeamButton = () => (
+    <Button
+      variant="fab"
+      StartIcon={Plus}
+      type="button"
+      href={`${WEBAPP_URL}/settings/teams/new?returnTo=${WEBAPP_URL}/teams`}>
+      {t("new")}
+    </Button>
+  );
   return (
     <Shell
       heading={t("teams")}
       hideHeadingOnMobile
       subtitle={t("create_manage_teams_collaborative")}
       CTA={
-        <Button
-          variant="fab"
-          StartIcon={Plus}
-          type="button"
-          href={`${WEBAPP_URL}/settings/teams/new?returnTo=${WEBAPP_URL}/teams`}>
-          {t("new")}
-        </Button>
+        !session.data?.user.organizationId ? (
+          <NewTeamButton />
+        ) : session.data?.user.isOrgAdmin ? (
+          <NewTeamButton />
+        ) : undefined
```

Comment:
```suggestion
    (!session.data?.user.organizationId || session.data?.user.isOrgAdmin) && <NewTeamButton />
```
Single line, let's try to avoid ternaries when more than 2 cases.